cfme/tests/automate/test_vmware_methods.py

```diff
@@ -86,7 +86,7 @@ def test_vmware_vimapi_hotadd_disk(
             3. After the button is created, it goes to a VM's summary page, clicks the button.
             4. The test waits until the capacity of disks is raised.
 
-    Bugzillas:
+    Bugzilla:
         1211627, 1311221
```

Comment:
Looking at Mike's PR #8675, you might want to keep one BZ on each line
```
Bugzilla:
        * 1211627
        * 1311221
````

Reply:
Thanks! I think I missed a few of those `BZ:` refs on rebase. I like the pattern on Mike's PR so I'll change that too

Reply:
Actually there are many such cases where `*` is absent, @mshriver do we want to include the `*` or leave it optional?

Reply:
Discussing with Mike on chat, he suggested without the `*`.